src/main/kotlin/app/revanced/patcher/extensions/InstructionExtensions.kt

```diff
@@ -11,7 +12,28 @@ import org.jf.dexlib2.builder.MutableMethodImplementation
 import org.jf.dexlib2.builder.instruction.*
 import org.jf.dexlib2.iface.instruction.Instruction
 
+
+class InstructionInserter(val method: MutableMethod, startIndex: Int) {
```

Comment:
I am not sure if this approach is good. Instead of allowing any extension function to be used from `InstructionExtensions`, you constrain it to two functions inside `InstructionInserter.` Instead, you should allow using any function from `InstructionExtensions.` I am not sure how this can be achieved through using function parameters.

Reply:
What do you mean by that?